### PR TITLE
Fixes #109 - Update speaker styles on index

### LINKED ISSUE #109: UXD: Update speaker styles on index

### PR DESCRIPTION
**Pull request by Ctannerweb**

### CODE REVIEW

**Comment by Ctannerweb** [comment]
@acacheung :eyes: has been through Q&A with Tim

File: views/index.erb [hunk 1]
```diff
@@ -43,38 +43,43 @@
   <h2 class="t-display">Speakers</h2>
   <ul class="speakers">
     <li class="speaker">
-      <a class="speaker__link" href="https://twitter.com/christse" title="Visit Chris Tse’s Twitter" target="_blank">
+      <a class="speaker__link" href="/speakers" title="Visit Chris Tse's Bio">
```
Comment (acacheung) [inline views/index.erb]: href="/speakers#chris-tse"

**Comment by acacheung** [comment]
can merge after the speaker links are corrected

**Comment by Ctannerweb** [comment]
@acacheung made changes to the layout of speakers from what Brain wanted to see.

File: views/index.erb [hunk 2]
```diff
@@ -41,44 +41,80 @@
...
+        <h3 class="speaker__name">John Kleinschmidt</h3>
+        <h4 class="speaker__position">Cure</h4>
+        <span class="t-link">Full Bio</span>
+      </a>
+    </li>
+    <li class="speaker">
+      <a class="speaker__link" href="/speakers#nathan-hammond" title="Visit Nathan Hammond's Bio">
+        <img class="speaker__image" alt="Nathan Hammond" src="speakers/nathan-hammond.jpg">
+        <h3 class="speaker__name">Nathan Hammond</h3>
+        <h4 class="speaker__position">Linkedin</h4>
```
Comment (acacheung) [inline views/index.erb]: LinkedIn

**Pull request merged**